src/main/archetype/core/src/main/java/core/models/MyProductTeaserImpl.java

```diff
@@ -80,7 +80,8 @@ public Boolean isShowBadge() {
 
     @Override
     public String getFormattedPrice() {
-        return productTeaser.getFormattedPrice();
+        // ProductTeaser.getFormattedPrice is deprecated
+        return null;
```

Comment:
Instead of returning `null`, you could return `productTeaser.getPriceRange().getFormattedFinalPrice();`.

Reply:
will change. Thanks